doc-site/docs/tutorials/notarized-tokens.md

```diff
@@ -0,0 +1,135 @@
+# Notarized Tokens
+
+In this tutorial, you’ll learn how to create and manage a basic token using the **Notarized Tokens (noto)** domain. Unlike the private storage example, these tokens can be transferred between nodes publicly, demonstrating how assets (e.g., “cash”) can be issued and tracked on the blockchain using Paladin.
```

Comment:
I think the key here is that the tokens:
- can be exchanged privately
- are controlled by (and always visible to) the notary party
- can be used to prove ownership at any given point by selectively disclosing information about tokens that you own